android/src/main/java/com/twilio/twilio_voice/AnswerJavaActivity.java

```diff
@@ -117,12 +117,12 @@ private void handleIncomingCallIntent(Intent intent) {
                     Log.d(TAG, "ending call" + activeCall != null ? "True" : "False");
                     if (activeCall == null) {
                         Log.d(TAG, "No active call to end. Returning");
-                        finish();
+                        finishAndRemoveTask();
```

Comment:
Interesting I tried to use 'finishAffinity()' but that did not seem to remove the app UI when the task finished.

Reply:
yeah it seems like even when all activities are closed, the task still remains.  I'm not sure why but this works!